src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3834,73 +3718,42 @@ namespace HVACMultiSpeedHeatPump {
         // SUBROUTINE INFORMATION:
         //       AUTHOR:          Lixing Gu, FSEC
         //       DATE WRITTEN:    June 2007
-        //       MODIFIED         na
-        //       RE-ENGINEERED    na
 
         // PURPOSE OF THIS SUBROUTINE:
         //  This routine will calcultes MSHP performance based on given system load
 
-        // METHODOLOGY EMPLOYED:
-        // na
-
-        // REFERENCES: na
-
-        // Using/Aliasing
-        using DXCoils::SimDXCoilMultiSpeed;
-        using Fans::SimulateFanComponents;
-        using HeatingCoils::SimulateHeatingCoilComponents;
-
-        // Locals
-        // SUBROUTINE ARGUMENT DEFINITIONS:
-
-        // SUBROUTINE PARAMETER DEFINITIONS:
-        //  INTEGER, PARAMETER  ::   On  = 1           ! Compressor on flag
-        //  INTEGER, PARAMETER  ::   Off = 2           ! Compressor off flag
-
-        // INTERFACE BLOCK SPECIFICATIONS
-        // na
-
-        // DERIVMS TYPE DEFINITIONS
-        // na
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int OutletNode;            // MSHP air outlet node
         int InletNode;             // MSHP air inlet node
         Real64 OutsideDryBulbTemp; // Outdoor dry bulb temperature [C]
         Real64 AirMassFlow;        // Air mass flow rate [kg/s]
-        int FanInletNode;          // MSHP air outlet node
-        int FanOutletNode;         // MSHP air inlet node
         Real64 SavePartloadRatio;
         Real64 SaveSpeedRatio;
         Real64 QCoilActual;  // coil load actually delivered returned to calling component
         Real64 MinWaterFlow; // minimum water flow rate
         Real64 ErrorToler;   // supplemental heating coil convergence tollerance
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3879]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const